### ISSUE TITLE
Develop

### ISSUE THREAD

**Issue by landsman** [neokoenig/jQuery-gridmanager#84]
Hi,

this is dead? :(

**Comment by neokoenig** [comment]
Not really. Just that I'm struggling to rewrite it with a better design methodology. The current stable release works, but for a newer version I was hoping to just make it ... 'better'. :)

**Comment by black-byte** [comment]
Any new informations about new version ?
